docs/releases/index.rst

```diff
@@ -11,34 +12,64 @@ Also be aware of the :ref:`important changes in the Translate Toolkit
 If you are upgrading Pootle, you might want to see some tips to ensure your
 :doc:`upgrade goes smoothly </server/upgrading>`.
 
+Final releases
+==============
+
+2.8 release
+-----------
+.. toctree::
+   :maxdepth: 1
+
+   2.8.0
```

Comment:
@julen good point.  I guess this is working on Django cause it's within their `dev` docs https://docs.djangoproject.com/en/dev/releases/

I'll figure something out, but ideally would like to push towards the Django way